galaxy.yml

```diff
@@ -0,0 +1,55 @@
+---
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: aalaesar
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: nextcloud
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 2.0.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
```

Comment:
Will this work since readme.md is being deleted from the root in this PR? Is it maybe missing a readme for the collection?

Reply:
The role's readme have been moved along it and  I created a new one for describing the collection. You should see a large diff for it